src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -208,8 +251,35 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check so that it matches default IntelliJ IDEA formatter configuration
- * (tested on v2018.2):
+ * Example:
+ * </p>
+ * <pre>
+ * import static java.awt.Button.A; // Violation; Should follow 'sortStaticImportsAlphabetically'
+ * import static java.lang.Math.PI;
+ * import static java.lang.Math.abs; // Ok, alphabetical case sensitive ASCII order, 'P' &lt; 'a'
+ * import java.lang.Math.sqrt; // Ok, Follow property 'Option' value 'above'
+ *
+ * import static java.io.File.createTempFile;
+ * import java.io.File;
+ *
+ * import java.io.IOException; // Violation; Should follow 'Group' property
+ *
+ * import org.albedo.*; // Violation; Should follow 'ordered' property *{comes after javax imports}
+ *
+ * import static javax.swing.WindowConstants.*;
+ * import javax.swing.JComponent;
+ * import org.apache.http.ClientConnectionManager; // Violation; Should follow 'separated' property
+ *
+ * import org.linux.apache.server.SoapServer;
+ *
+ * import com.neurologic.http.HttpClient;
+ * import com.neurologic.http.impl.ApacheHttpClient;
```

Comment:
Are these ok or violation?

Reply:
ok

Reply:
i write "// Ok" at the end of line 273, 275, 276.

Reply:
Resolved